llvm/test/DebugInfo/CAS/AArch64/multiple_cus.ll

```diff
@@ -0,0 +1,106 @@
+; RUN: llc -cas-friendly-debug-info -O0 --filetype=obj --cas-backend --cas=%t/cas --mccas-casid --mtriple=arm64-apple-darwin %s -o %t/multiple_cus.id 2>&1 | FileCheck %s --allow-empty
+; CHECK-NOT: error in backend: CASBackend output round-trip verification error
```

Comment:
Is there anything positive we could check for, too?

Reply:
Maybe we can add an llvm-cas-dump test to check if we see two DIETopLevelRefs and two distinctdatarefs under the debug_line section?